rdt/transformers/__init__.py

```diff
@@ -25,6 +25,14 @@
     transformer.__name__: transformer
     for transformer in BaseTransformer.__subclasses__()
```

Comment:
could call `BaseTransformers.get_subclasses()` here, so `TRANSFORMERS` is a list of all transformers.

Reply:
Also could we add `'GaussianCopulaTransformer` to the `__all__` list above?

Reply:
It's already there